lib/network_utils.ml

```diff
@@ -1163,3 +1200,196 @@ module Ethtool = struct
 		if options <> [] then
 			ignore (call ~log:true ("-K" :: name :: (List.concat (List.map (fun (k, v) -> [k; v]) options))))
 end
+
+module Sriov = struct
```

Comment:
Note that Network_utils (this file) is organised into sub-modules that are based on Linux tools/features (see comment below on `make_vf_conf_internal`). `Sriov` doesn't quite fit this model. It looks like most of the functions below belong inside some of the existing modules above, or in new ones, or should just move to the top-level in network_server.ml.